compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -224,6 +243,16 @@ namespace jank::read::lex
     return !(*this == rhs);
   }
 
+  bool big_decimal::operator==(big_decimal const &rhs) const
+  {
+    return number_literal == rhs.number_literal;
+  }
+
+  bool big_decimal::operator!=(big_decimal const &rhs) const
+  {
+    return !(*this == rhs);
+  }
+
```

Comment:
These fns don't need to exist. The C++ compiler will give us the operators for free, since all members already implement them.

Reply:
You need to remove its declaration in the header too, brother!